source/guide_soju.rst

```diff
@@ -0,0 +1,139 @@
+.. highlight:: console
+
+.. author:: apoxa <https://github.com/apoxa>
+
+.. tag:: lang-go
+.. tag:: irc
+.. tag:: bouncer
+
+.. spelling::
+    Soju
+    soju
+    upstreams
+
+##########
+Soju
+##########
+
+.. tag_list::
+
+soju_ is a user-friendly IRC bouncer which supports multiple users, multiple clients for a single user with backlog synchronization and multiple upstreams via a single IRC connection to the bouncer.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Firewall Ports <basics-ports>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+All relevant legal information can be found here:
+
+  * https://git.sr.ht/~emersion/soju/tree/master/item/LICENSE
+
+Prerequisites
+=============
+
+.. include:: includes/open-port.rst
+
+Installation
+============
+
+Clone the source into your home directory and enter the directory:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://git.sr.ht/~emersion/soju
+ [isabell@stardust ~]$ cd ~/soju
```

Comment:
the cd command should be moved to the next code-block

Reply:
Yea, that make sense in the overall layout :)